schema/INDIVIDUAL_NATIONAL_PARTY_HEADQUARTERS_ACCOUNT.json

```diff
@@ -0,0 +1,577 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/INDIVIDUAL_NATIONAL_PARTY_HEADQUARTERS_BUILDINGS_ACCOUNT.json",
+    "version": "v0.0.0.0",
+    "title": "FEC Individual National Party Headquarters Buildings Account",
+    "description": "Individual National Party Headquarters Buildings Account (17)",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_type_identifier",
+        "transaction_id",
+        "entity_type",
+        "contributor_last_name",
+        "contributor_first_name",
+        "contributor_street_1",
+        "contributor_city",
+        "contributor_state",
+        "contributor_zip",
+        "contribution_date",
+        "contribution_amount",
+        "contribution_aggregate",
```

Comment:
looks like cpd should be here

Reply:
cpd and memo_code added to required field list.